integrations/github.rst

```diff
@@ -0,0 +1,82 @@
+* Rebuild the environment when pushing new code to GitHub.
+* Delete the environment when merging a pull request.
+
+Setup
+-----
+
+1. Generate a token
+^^^^^^^^^^^^^^^^^^^
+
+To integrate your project with an existing GitHub repository, you first need to generate a token on your GitHub user profile. Go to your Settings, then select ``Developer settings`` and click ``Personal access tokens``. Here you can `Generate a new token <https://github.com/settings/tokens/new>`_.
+
+Give it a description and then ensure the token has the following scopes:
+
+* To integrate with public repositories: ``public_repo``
+* To integrate with your own private repositories: ``repo``
+* To integrate with your organization's private repositories: ``repo``
+  and ``read:org``
+
+Copy the token and make a note of it (temporarily).
+
+Note that for the integration to work, your GitHub user needs to have permission to push code to the repository.
+
+2. Enable the integration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enable the GitHub integration as follows:
+
+.. code-block: bash
+
+   $ symfony integration:add --type=github --repository=USER/REPOSITORY
+
+where
+
+* ``USER`` is your GitHub user name
+* ``REPOSITORY`` is the name of the repository in GitHub (not the git address)
+
+Note that if your repository belongs to an organization, use
+``--repository=ORGANIZATION/REPOSITORY``.
+
+Optional parameters:
+
+* ``--fetch-branches``: Track and deploy branches (true by default)
+* ``--prune-branches``: Delete branches that do not exist in the remote GitHub
+  repository (true by default)
+* ``--build-pull-requests``: Track and deploy pull-requests (true by default)
+* ``--build-pull-requests-post-merge``: ``false`` to have SymfonyCloud build
+  the branch specified in a PR. `true` to build the result of merging the PR
+  (``false`` by default)
+* ``--pull-requests-clone-parent-data``: Set to ``false`` to disable cloning of 
+  parent environment data when creating a PR environment, so each PR
+  environment starts with no data. (``true`` by default)
+
+3. Add the webhook
+^^^^^^^^^^^^^^^^^^
+
+#. Copy the hook URL shown in the previous step.
+#. Go to your GitHub repository and click Settings, select the Webhooks and Services tab, and click Add webhook.
+#. Paste the hook URL, choose ``application/json`` for the content type, choose
+   "Send me everything" for the events you want to receive, and click "Add 
+   webhook".
+
+You can now start pushing code, creating new branches or opening pull requests directly on your GitHub repository.
+
+Types of environments
+---------------------
+
+Environments based on GitHub **pull requests** will have the correct 'parent'
+environment on SymfonyCloud; they will be activated automatically with a copy of
+the parent's data.
+
+However, environments based on (non-pull-request) **branches** cannot have
+parents; they will inherit directly from ``master`` and start inactive by
```

Comment:
Perhaps consider using _default branch_ instead of `master`?

In case you haven't noticed, GitHub and a lot of other people are moving away from using `master` as a default branch name.

Also see https://localheinz.com/blog/2020/06/12/black-lives-matter/.